test/expr/postfix/dot/init_ref_delegation.swift

```diff
@@ -549,10 +549,9 @@ struct MultipleMemberAccesses {
 
 func sr10670() {
   struct S {
-    init(_ x: inout String) {}
-    init(_ x: inout [Int]) {}
+    init(_ x: inout String) {} // expected-note {{candidate expects value of type 'String' for parameter #1}}
```

Comment:
@jrose-apple Should we only mention the type after `inout` in situations like this or use something like `address of value of type 'String'`?

Reply:
"Address" would be wrong. I'm not sure we have a general user-facing term for "lvalue". @natecook1000, @krilnon, any ideas?

Alternately, just go with "candidate expects inout value of type 'String'" even though that doesn't 100% make sense on close inspection.

Reply:
Thanks! I'll change it to add `inout` for now.

Reply:
Can you use "expects an in-out value" instead? `inout` is the keyword, but we hyphenate when using it in a sentence.

Reply:
Absolutely!